Sources/Orbit/Support/Environment Keys/BackgroundColorKey.swift

```diff
@@ -0,0 +1,68 @@
+import SwiftUI
+
+/// Stores background default and active color used for Orbit components that contain background, such as `Card` or `Tile`.
+public struct BackgroundColor {
+    
+    public let inactive: any ShapeStyle
+    public let active: any ShapeStyle
+    
+    @ViewBuilder public var inactiveView: some View {
+        switch inactive {
+            case let color as Color:            color
+            case let color as LinearGradient:   color
+            case let color as RadialGradient:   color
+            default:                            EmptyView()
+        }
+    }
+    
+    @ViewBuilder public var activeView: some View {
+        switch active {
+            case let color as Color:            color
+            case let color as LinearGradient:   color
+            case let color as RadialGradient:   color
+            default:                            EmptyView()
+        }
+    }
+}
+
+struct BackgroundColorKey: EnvironmentKey {
+    static var defaultValue: BackgroundColor?
+}
+
+public extension EnvironmentValues {
+
+    /// A background color stored in a view’s environment, used for Orbit components that contain background, such as `Card` or `Tile`.
+    var backgroundColor: BackgroundColor? {
+        get { self[BackgroundColorKey.self] }
+        set { self[BackgroundColorKey.self] = newValue }
+    }
+}
+
+public extension View {
```

Comment:
Hmm... won't this be too similar to SwiftUI's existing `.background`?

Reply:
It is, but I did not found a better name for it. It represents the `background` that would be used inside the component.
Another possibility would be `.backgroundShape`.

It is now matching with our `textColor` which is different to the `foregroundColor`.